### PR TITLE
:pencil: Copy editing

### PR DESCRIPTION
**Pull request by jefftriplett**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -195,18 +195,18 @@ When updating to a new version, it is recommended to follow the following steps:
...
+2. A release that removes the backward compatible changes and removes the deprecation warning. This release will be tagged with a major version bump, e.g., `0.2.0` to `1.0.0`.
 
 This is unlikely to happen often, but it is important to keep in mind when updating.
 
 A major release does not necessarily mean that there are breaking changes or ones involving models and migrations. You should always check the [changelog](CHANGELOG.md) and a version's release notes for more information.
 
 ## Usage
 
-Once your Django project is configured to use `email_relay.backend.RelayDatabaseEmailBackend` as it's `EMAIL_BACKEND`, sending email is as simple as using Django's built-in ways of sending email, such as the `send_mail` method:
+Once your Django project is configured to use `email_relay.backend.RelayDatabaseEmailBackend` and its `EMAIL_BACKEND`, sending email is as simple as using Django's built-in ways of sending email, such as the `send_mail` method:
```
Comment (joshuadavidthomas) [inline README.md:209]: this one should be "as its"

File: README.md [hunk 2]
```diff
@@ -72,9 +72,9 @@ docker run -d \
   ghcr.io/westerveltco/django-email-relay:latest
 ```
 
-It is recommended to pin to a specific version, though if you prefer you can ride the lightning by always pulling the `latest` image.
+It is recommended to pin to a specific version, though if you prefer, you can ride the lightning by always pulling the `latest` image.
 
-See the documentation [here](#configuration) for general information about configuring `django-email-relay`, [here](#relay-service-1) for general information about configuring the relay service, and [here](#docker-1) for information specifically related to configuring the relay service as a Docker container.
+See the documentation [here](#configuration) for general information about configuring `django-email-relay`, [here](#relay-service-1) for available information about configuring the relay service, and [here](#docker-1) for information specifically related to configuring the relay service as a Docker container.
```
Comment (joshuadavidthomas) [inline README.md:77]: Not sure about this one. Could just shorten both uses of "general information" to "information"

**Review by joshuadavidthomas — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jefftriplett) [reply README.md:209]:
```suggestion
Once your Django project is configured to use `email_relay.backend.RelayDatabaseEmailBackend` as its `EMAIL_BACKEND`, sending email is as simple as using Django's built-in ways of sending email, such as the `send_mail` method:
```

File: README.md (hunk 2 above)
Comment (jefftriplett) [reply README.md:77]:
```suggestion
See the documentation [here](#configuration) for general information about configuring `django-email-relay`, [here](#relay-service-1) for information about configuring the relay service, and [here](#docker-1) for information specifically related to configuring the relay service as a Docker container.
```

**Review by joshuadavidthomas — APPROVED** [review]

**Pull request merged**